src/main/java/org/traccar/protocol/Gl200TextProtocolDecoder.java

```diff
@@ -1074,6 +1077,11 @@ private Object decodeIgn(
         return position;
     }
 
+    private boolean isVirtualIgnitionMode(String type, String model) {
```

Comment:
I don't think we need a separate function for this. Also what does `type.contains("VG")` mean?